proto/minder/v1/minder.proto

```diff
@@ -2671,3 +2671,22 @@ message EvaluationHistoryAlert {
     // the structure and contents are alert specific, and are subject to change.
     string details = 2;
 }
+
+message EntityInstance {
+    // id is the unique identifier of the entity.
+    string id = 1;
+
+    // context is the context in which the entity is evaluated.
+    ContextV2 context = 2;
+
+    // name is the name of the entity.
+    string name = 3;
+
+    // type is the type of the entity.
+    // DISCUSSION: If we're aiming for a BYO entity type, we should probably
+    // have this be a string, and have the user provide the type.
+    Entity type = 4;
```

Comment:
I think this is a good goal. Since this is a public API, do you think it makes sense to have the attribute typed to a string from the start so that we don't have to deprecate or change the type later? OTOH until the type is used in a public API we might as well be a little looser with the compatibility.

Reply:
sounds good to me. So, should we keep it as is for now?

Reply:
yes, I think for now it's safer to have a typed attribute and the migration seems easy.